webshot/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
any reason why this is here and not at the root level?

Reply:
we've kept the app and api gitignores distinct (without agreeing explicitly on this, but I guess it's an unspoken agreement to avoid bickering between FE and BE 😁 ) and I have kept this distinct for webshot too - I see you've added tf ignores in the root level one, and I agree with that - I'd stick to per-"application module" gitignores (and top level for infra and editor/IDE rubbish, which is more or less what is now, except a couple of python and node-specific excludes, though they're quite ubiquitous in any case so top level also makes sense), but if you prefer this could go as `/webshot/dist` in the top level gitignore too.